.travis.yml

```diff
@@ -117,9 +109,31 @@ jobs:
         - dist/dvc*.whl
         - dist/dvc*.tar.gz
       skip_cleanup: true
-      on:
-        tags: true
-        repo: iterative/dvc
+  - name: Snapcraft snap
+    stage: check  # TODO: remove before merge
```

Comment:
@efiop I think this is ready for merge now. Please remove this line before, though!

Reply:
@casperdcl Agreed. Please address that README comment and remove this line 🙂